
When altivec is disabled there is no swScale_altivec ...

Additionally cleaned up the else case a bit.

  - Rene Rebe <rene@exactcode.de>

RUN_CPUDETECT='0' failed, cut down to the bare minimum.

  - Dimitar Zhekov <jimmy@is-vn.bg>

--- ./postproc/swscale.c.orig	2004-04-26 22:38:17.000000000 +0300
+++ ./postproc/swscale.c	2004-05-08 12:53:21.000000000 +0300
@@ -1344,10 +1344,11 @@
 
 #else
 #ifdef ARCH_POWERPC
+#ifdef HAVE_ALTIVEC
 	if(flags & SWS_CPU_CAPS_ALTIVEC)
 	  return swScale_altivec;
 	else
-	  return swScale_C;
+#endif
 #endif
 	return swScale_C;
 #endif
